src/main.rs

```diff
@@ -67,6 +67,17 @@ fn nand(a: bool, b: bool) -> bool {
     !(a && b)
 }
 
+fn get_link_by_id(store: &mut unit::Store<usize, _>, id: usize) -> Result<Link<usize>, Error<usize>> {
+    // `any` constant denotes any link
+    let any = store.constants().any;
+
+    store.each_iter([id, any, any]).for_each(|link| {
+        println!("{link:?}");
+    });
+
+    Ok(())
+}
+
```

Comment:
This function only prints link, but should return it instead from get_link_by_id.